ref/feature/spnego/examples.adoc

```diff
@@ -0,0 +1,27 @@
+== Examples
+
+Open Liberty supports many common SPNEGO scenarios and configurations.
+
+=== Limit SPNEGO authentication 
+You can filter HTTP requests to require SPNEGO authentication for only certain requests, web applications, hosts, or user agents.
+
+The following example configures the following things:
+<1> SPNEGO authentication is used for any requests that are received containing the `example.com` hostname for resources within the `protectedApp` web application.
```

Comment:
"SPNEGO authentication is used for any requests that are received containing the `example.com` hostname for resources within the `protectedApp` web application."
to 
"SPNEGO authentication is used for any requests received that contain the `example.com` hostname for resources within the `protectedApp` web application."

Reply:
Updated